examples/webview/webview/app.py

```diff
@@ -92,6 +106,9 @@ def startup(self):
             on_webview_load=self.on_webview_load,
             style=Pack(flex=1),
         )
+        # activate web navigation filtering on supported platforms
+        if getattr(self.webview._impl, "SUPPORTS_ON_NAVIGATION_STARTING", True):
```

Comment:
Good instinct, but it's fine for the example to raise a warning if a feature isn't available.